### ISSUE TITLE
Discord Support

### ISSUE THREAD

**Issue by Thurrax** [FezVrasta/ark-server-tools#961]
Hey there, im using the Arkmanger along time already and i tought about a integration with the Discord Webhook. Like when Arkmanager detects a new Update or Mods need to be Update or even when the Server does Updates. In general that the arkmanager posts via webhook into discord.

**Comment by klightspeed** [comment]
813ae33 adds Discord notification support.

**Comment by Thurrax** [comment]
Dang that was fast. When can i count on an update?

**Comment by klightspeed** [comment]
v1.6.41 includes Discord notification support.  Refer the `Notification` section of the example `arkmanager.cfg`, and the `discordWebhookURL` and `notify*` options in the readme.

**Comment by B00um** [comment]
Hi there!
Is there a way to send only some notifications?
For example, sending notifications when server is starting but not when shutting down?

Maybe this is something obvious but I do not undestand how to do such a thing. 
I tried to disable unwanted notifications or setting them empty but in this cas the default ones are sent.

Thanks for your help ^^

![capture](https://user-images.githubusercontent.com/33394804/41536319-92c04348-7305-11e8-841d-5bb2a1aea297.PNG)

**Comment by kayzore** [comment]
Hi !

Thanks for this great tool !

I have exactly the same problem. I don't receive the shutdown message...
Have you got a solution or this is an actual issue ?

Kayzore.

**Comment by OverMind79** [comment]
Hello

when i enter : 
arkmanager restart
i become the shutting down message after the the server starting message but my server starts normal after some minutes he is online but i in Discord comes no ServerUp message .

have someone an idee?

**Comment by kayzore** [comment]
Hi everybody,

Today, I cloned my old installation on a new server. On this server, I have two instances of Ark and it works well! But I have a problem with the discord integration... The channel receives well each notification except the message serverUp.

It's almost the same problem as OverMind79 ...

I tried to move the configuration of notifications in the arkmanager.cf (which is the configuration on my old server and with it the message is well received ...)

Do you have a solution to this problem?

**Comment by klightspeed** [comment]
Is `notifyMsgServerUp` set in your arkmanager instance config?

Are the shutting down and starting messages delivered?

Is `server is up` logged to the `arkserver.log`?

**Comment by kayzore** [comment]
Thank you for your reply

I tried both types of settings (in the arkmanager.cfg and in my instance.cfg) without success ...

Yes, I only get the starting and shutting down message

And not in the arkserver.log the server is up messagee is not logged ...
It's really strange...

**Comment by klightspeed** [comment]
What does `arkmanager status @instance` show?  The server up check in that is the same as the server up check that logs `server is up` and sends the notification.

**Comment by kayzore** [comment]
After a few minutes of waiting, the arkmanager status @instance command displays the server is up and listening and I can connect and play on it, but the serverUp message is not sent to my channel discord

**Comment by klightspeed** [comment]
Do you have `arkAlwaysRestartOnCrash` set in your config?  This currently bypasses the online check.  I'll sort out a fix to perform the online check even when `arkAlwaysRestartOnCrash` is set.

**Comment by kayzore** [comment]
Yes indeed, I had put this option in my instance ... After commenting on it I receive the serverUp message !
Waiting for a fix, maybe you should indicate it in the readme ;)

Thanks a lot for your help !

**Comment by klightspeed** [comment]
v1.6.42 contains the fix

**Comment by kayzore** [comment]
Big thanks for your reactivity !

**Comment by kayzore** [comment]
Hi, I made the update in 1.6.42 and I think I noticed a bug, now when I restart the server I receive the start message and directly after the serverUp message while the command status indicates listenning no
